migrate.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS `vehicle_financing` (
+    `vehicleId` int(11) NOT NULL,
+    `balance` int(11) DEFAULT NULL,
+    `paymentamount` int(11) DEFAULT NULL,
+    `paymentsleft` int(11) DEFAULT NULL,

```

Comment:
We could probably switch this to tinyint as paymentsleft would never go above 255

Reply:
It should be alright yeah, but if it breaks anything i'll blame you! 😆